app/src/newdocs/content/streams.mdx

```diff
@@ -12,7 +15,55 @@ import docsStyles from '$newdocs/components/DocsLayout/docsLayout.pcss'
 <ScrollableAnchor id="work-with-streams-in-core"><div>
 
 ## Work with streams in Core
---content--
+Core provides you an easy way to manage your streams. Streams can be created, edited and deleted through our intuitive UI.
+
+### Overview of your streams
+Head out to Streams section to see an overview of your streams.
+
+<img src={streamListing} />
+
+Stream listing shows statuses of your streams at a glance.
+Last received event timestamp will be shown per stream as well as traffic light for your stream status.
+
+| Color        | Status                           |
+| ------------ | -------------------------------- |
+| Green        | Stream is receiving data         |
+| Red          | Stream is not receiving data     |
+
+You can click the stream name to open the edit view. By clicking the three dots in the last column of the table a stream can be e.g. shared.
+Also a new stream can be created by clicking the blue button in the top right section of the page.
+
+### Editing a stream
+Stream editor provides basic tools for setting up your stream.
+
+<img src={streamSettings} />
```

Comment:
Looks like there's a screenshot artefact on inside the historical data section - maybe recapture this or reach out to Ross as he's keen to help out for things like this.

Reply:
Good spot! I recaptured this one.